.travis.yml

```diff
@@ -6,11 +6,14 @@ env:
 rvm:
   - 2.0
   - 2.1
-  - 2.2.1
+  - 2.2.7
```

Comment:
We should add 2.2.2, since I've seen cases where stuff worked in 2.2.7, but not in 2.2.2.

Reply:
Okay, it's been re-added.